src/lib/dom.js

```diff
@@ -4,12 +4,12 @@ export function div(className) {
   return div;
 }
 
-const elMatches =
-  Element.prototype.matches ||
-  Element.prototype.webkitMatchesSelector ||
-  Element.prototype.msMatchesSelector;
-
 export function matches(element, query) {
+  const elMatches =
```

Comment:
It's better not to calculate `elMatches` every time `matches()` is called. Can you revert it to a root-level variable and just add a check for `Element`?

Reply:
Good point, fixed it